src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -753,6 +753,55 @@
             sb.Length.Should().Be(0, $"test cases failed : {Environment.NewLine}{sb}");
         }
 
+        [Fact]
+        public void AnalyzeCommand_Traces_ExternalValueShouldNotThrowException()
```

Comment:
>AnalyzeCommand_Traces_ExternalValueShouldNotThrowException [](http://example.com/codeflow?start=20&length=58)

This test is added to show that after fix, the PdbLoad value will not throw any error.